install.sh

```diff
@@ -12,28 +14,29 @@ apt-get update
 
 echo "================= Adding some global settings ==================="
 mv gbl_env.sh /etc/profile.d/
-mkdir -p $HOME/.ssh/
-mv config $HOME/.ssh/
+mkdir -p "$HOME/.ssh/"
+mv config "$HOME/.ssh/"
 mv 90forceyes /etc/apt/apt.conf.d/
-touch $HOME/.ssh/known_hosts
+touch "$HOME/.ssh/known_hosts"
 
 echo "================= Installing basic packages ==================="
 apt-get install -y \
-    sudo  \
     build-essential \
     curl \
     gcc \
+    gettext \
```

Comment:
Added `gettext`. The rest of the diff is because I sorted this list alphabetically.